core/src/main/scala/org/broadinstitute/dsde/rawls/config/MultiCloudWorkspaceConfig.scala

```diff
@@ -0,0 +1,45 @@
+package org.broadinstitute.dsde.rawls.config
+
+import com.typesafe.config.Config
+import org.broadinstitute.dsde.rawls.util
+import org.broadinstitute.dsde.rawls.util.ScalaConfig._
+
+import scala.concurrent.duration._
+import scala.language.postfixOps
+
+
+// TODO this data will be pulled from the spend profile service, hardcoding in conf until that svc is ready
+final case class MultiCloudWorkspaceConfig(multiCloudWorkspacesEnabled: Boolean,
+                                           cloudContextPollTimeout: FiniteDuration,
+                                           azureConfig: Option[AzureConfig])
+
+final case class AzureConfig(spendProfileId: String,
+                             azureTenantId: String,
+                             azureSubscriptionId: String,
+                             azureResourceGroupId: String)
+
+
+case object MultiCloudWorkspaceConfig {
+  def apply[T <: MultiCloudWorkspaceConfig](conf: Config): MultiCloudWorkspaceConfig = {
+    val azureConfig: Option[AzureConfig] = conf.getConfigOption("multiCloudWorkspaces.azureConfig") match {
+      case Some(azc) => Some(AzureConfig(
+        azc.getString("spendProfileId"),
+        azc.getString("tenantId"),
+        azc.getString("subscriptionId"),
+        azc.getString("resourceGroupId")
```

Comment:
Looks like there is some missing test coverage?

Reply:
Added a small test for this class.